src/gcp_scanner/crawl.py

```diff
@@ -857,9 +853,9 @@ def get_endpoints(project_id: str,
     request = service.services().list(producerProjectId=project_id)
     while request is not None:
       response = request.execute()
-      for service_entry in response.get("services", []):
-        endpoints_list.append(service_entry)
-
+      endpoints_list = [service_entry 
+                        for service_entry in response.get("services", [])]
+      
```

Comment:
Whitespace

Reply:
I have made the requested changes in the latest PR. Check and let me know if any changes required.